tests/xtro-sharpie/iOS-MapKit.ignore

```diff
@@ -9,3 +9,6 @@
 !missing-field! MKAnnotationCalloutInfoDidChangeNotification not bound
 !missing-protocol-conformance! MKUserLocation should conform to MKAnnotation
 !missing-protocol-member! MKOverlay::coordinate not found
+
+## Getting "[FAIL] Selector not found for MapKit.MKMapItem : encodeWithCoder:" when 'MKMapItem' implements 'NSSecureCoding' on 32-bit iOS
+!missing-protocol-conformance! MKMapItem should conform to NSSecureCoding (defined in 'MKMapItemSerialization' category)
```

Comment:
looks like the previous case, it must be handled like a version change (not an arch specific change)
keep it mind xtro uses the 64bits .pch (from sharpie) to applies the rules

Reply:
oops, I reviewed the individual commits and some comments are not showing, see the _previous_ in https://github.com/xamarin/xamarin-macios/pull/3187/commits/0f00cb1e1424ff4665069b433c673bf27e16975c